### DISCUSSION TITLE
Not changing Slider on "keyup"!!!

### DISCUSSION THREAD

**Discussion by ashokavarthanan** [andreruffert/rangeslider.js#363] [Q&A]
Hi. Excellent your work, thanks for it. 
I have problem when using "up" or "down" Key Slider is not changing.
I using this URL "https://codepen.io/andreruffert/pen/meeOav";

But in normal <html> type "range" is working fine.
Thanks.

**Comment by lisardo2** [comment]
Your example does work but you have give the handle focus before you use the up and down arrow keys. Click in the bottom area, than tab so the handle get focus and you can use the up and down arrow keys. The page up and page down keys work too.

**Comment by ashokavarthanan** [comment]
Thanks for replay.
it works when i added "focus"  jQuery method.  
"$element.focus();"
